src/utils.js

```diff
@@ -1,5 +1,5 @@
 export { delay, noop, is, deferred, arrayOfDeferred, createMockTask, cloneableGenerator } from './internal/utils'
 
 export { SAGA_ACTION, TASK } from './internal/symbols'
-export { asEffect } from './internal/io'
+export { asEffect, inspection } from './internal/io'
```

Comment:
would creating a new entry point - `redux-saga/inspection` (like `/effects` and `/utils`) make sense for you here?

I'd like also to split `redux-saga` in few packages https://github.com/redux-saga/redux-saga/issues/1213 . However for the time being `redux-saga` would mainly stay the same, reexporting other packages etc. The reason for the change is just being able to install subpackages in other projects when the `redux-saga` itself is not needed. In such case - should `inspection` become other official subpackage or do you see it as part of the core?

Reply:
Thanks for the feedback! I've taken a stab at reorganizing it based on your recommendations, but as I don't have a lot of JavaScript experience, it may not be what you're looking after (I didn't know what tree-shaking was, for example, and still am a bit fuzzy on the concept after reading about it).  Lemme know!